ansible/roles/rails/templates/database.yml

```diff
@@ -1,5 +1,5 @@
 {{rails_env}}:
-  adapter: postgresql
+  adapter: {{database_adapter}}
   encoding: unicode
```

Comment:
should you just go ahead and allow customization of this? it's the last thing left in here that's not a variable